### PR TITLE
Run the solr java script in the background.

### PR DESCRIPTION
**Pull request by nirgn975**
#1

@RoySegall With that fix, solr will work on the background.

Maybe it's good to add a readme file what will explain that and will explain how to close it (like `top | grep java` and then `kill <pid>`.

### CODE REVIEW

File: solr.sh [hunk 1]
```diff
@@ -23,4 +23,4 @@ cd -
 # Start the server.
 cd solr/example/
 
-java -jar start.jar
+java -jar start.jar > /dev/null 2>&1 < /dev/null &
```
Comment (RoySegall) [inline solr.sh]: We would need to provide an argument for that. And also update the readme file.

10x.

**Comment by nirgn975** [comment]
@RoySegall I created a readme file, and add an `if else` for that background option.

**Comment by nirgn975** [comment]
@RoySegall Now it's really 10x.

File: solr.sh [hunk 2]
```diff
@@ -23,4 +23,25 @@ cd -
...
 
+#
+while test $# -gt 0; do
+  case "$1" in
+    -h|--help)
+      echo " "
+      echo "solr-script by roy segall"
+      echo " "
+      echo "options:"
+      echo "-h, --help                      show brief help"
```
Comment (RoySegall) [inline solr.sh]: Remove the help. Look redundent.

**Comment by nirgn975** [comment]
@RoySegall Ready for review.

**Pull request merged**